static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -93,9 +90,26 @@ function getHoverBody(
 
 // NOTE: Will be adding switch cases as more contexts require headers.
 function getHoverHeader(dataRow: EventData, contextType: ContextType) {
-  return isReleaseContext(contextType) ? (
-    <VersionHoverHeader releaseVersion={dataRow.release} />
-  ) : null;
+  switch (contextType) {
+    case ContextType.RELEASE:
+      return (
+        <HoverHeader
+          title="Release"
```

Comment:
This and 105 need `t()`
```suggestion
          title={t('Release')}
```

Reply:
good catch!